src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTableTabs.tsx

```diff
@@ -0,0 +1,184 @@
+                retryLoadEntries={() => retryLoadEntries?.(EntryScope.Widget)}
+                isErrorMessage={mapErrors?.[EntryScope.Widget]}
+                isLoading={mapLoaders?.[EntryScope.Widget]}
+                workbook={workbook}
+                onRenameEntry={onRenameEntry}
+                onDeleteEntry={onDeleteEntry}
+                onDuplicateEntry={onDuplicateEntry}
+                onCopyEntry={onCopyEntry}
+                createEntryBtn={<CreateEntry className={b('controls')} scope={EntryScope.Widget} />}
+                clearView={clearViewWidget}
+                onShowRelatedClick={onShowRelated}
+            />
+            {showDataEntities && (
+                <MainTabContent
+                    chunk={datasetChunk}
+                    actionCreateText={i18n('action_create-dataset')}
+                    title={i18n('title_datasets')}
+                    actionType={CreateEntryActionType.Dataset}
+                    isShowMoreBtn={Boolean(
+                        datasetChunk?.length > 0 && mapTokens?.[EntryScope.Dataset],
+                    )}
+                    loadMoreEntries={() => loadMoreEntries?.(EntryScope.Dataset)}
+                    retryLoadEntries={() => retryLoadEntries?.(EntryScope.Dataset)}
+                    isErrorMessage={mapErrors?.[EntryScope.Dataset]}
+                    isLoading={mapLoaders?.[EntryScope.Dataset]}
+                    workbook={workbook}
+                    onRenameEntry={onRenameEntry}
+                    onDeleteEntry={onDeleteEntry}
+                    onDuplicateEntry={onDuplicateEntry}
+                    onCopyEntry={onCopyEntry}
+                    onShowRelatedClick={onShowRelated}
+                />
+            )}
+            {showDataEntities && (
+                <MainTabContent
+                    chunk={connChunk}
+                    actionCreateText={i18n('action_create-connection')}
+                    title={i18n('title_connections')}
+                    actionType={CreateEntryActionType.Connection}
+                    isShowMoreBtn={Boolean(
+                        connChunk?.length > 0 && mapTokens?.[EntryScope.Connection],
+                    )}
+                    loadMoreEntries={() => loadMoreEntries?.(EntryScope.Connection)}
+                    retryLoadEntries={() => retryLoadEntries?.(EntryScope.Connection)}
+                    isErrorMessage={mapErrors?.[EntryScope.Connection]}
+                    isLoading={mapLoaders?.[EntryScope.Connection]}
+                    workbook={workbook}
+                    onRenameEntry={onRenameEntry}
+                    onDeleteEntry={onDeleteEntry}
+                    onDuplicateEntry={onDuplicateEntry}
+                    onCopyEntry={onCopyEntry}
+                    onShowRelatedClick={onShowRelated}
+                />
+            )}
+            {/* todo: move to other place everything below after update */}
+            {availableScopes?.includes(EntryScope.Report) && (
```

Comment:
How are we gonna add `Report` tab without `EntryScope.Report` ?

Reply:
This is customizable component from registry. In next step we will add custom definition of this component to registry in closed source